fcrepo/fcrepo
# Throw exceptions for managed types when in RELAXED mode

**JIRA Ticket**: https://jira.lyrasis.org/browse/FCREPO-3645

# What does this Pull Request do?
To more closely mimic Fedora 5s behaviour this alters Fedora 6 to throw an exception when a user adds a managed type in their RDF when the server is in relaxed mode. Previously it just ignored the triple.

# How should this be tested?

1. Start Fedora in relaxed mode. ie. (`fcrepo.properties.management=relaxed`)
1. Create a resource a new resource.
1. GET the resource without server managed properties, or remove them from the result to make it easier to see what is happening.
1. Add a server managed type to the body, like `<> <http://www.w3.org/1999/02/22-rdf-syntax-ns#type> <http://www.w3.org/ns/ldp#NonRDFSource>`
1. Try to PUT this back to the server.
1. In `main` you will succeed but nothing changes.
1. In this PR you will get a 409 Conflict.

# Interested parties
@fcrepo/committers, specifically @bbpennel
